tensorflow/stream_executor/cuda/cuda_gpu_executor.cc

```diff
@@ -195,7 +195,7 @@ static std::string GetBinaryDir(bool strip_exe) {
   _NSGetExecutablePath(nullptr, &buffer_size);
   char unresolved_path[buffer_size];
   _NSGetExecutablePath(unresolved_path, &buffer_size);
-  CHECK_ERR(realpath(unresolved_path, exe_path) ? 1 : -1);
+  // CHECK_ERR(realpath(unresolved_path, exe_path) ? 1 : -1);  //Tom: to sove "use of undeclared identifier 'CHECK_ERR'" on macOS
```

Comment:
Can you find out why CHECK_ERR isn't available here on macOS?

Reply:
Where is the macro CHECK_ERR supposed to be defined? I searched the whole TF project but cannot find it anywhere else. 

An interesting thing is that the part of code is to be compiled for macOS only. I wonder if is it really tested on macOS or not :)